shared/fs/really-delete/index.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import {Box, ConfirmModal, HeaderOnMobile, Icon, MaybePopup, ProgressIndicator} from '../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
```

Comment:
Mind using `import * as Kb` and `import * as Styles` here? We prefer this style for common-adapters and Styles in new code.